### DISCUSSION TITLE
2010/08/05/google-wave%E5%B0%B1%E8%A6%81%E8%AF%B4%E6%8B%9C%E6%8B%9C%E4%BA%86

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#292] [Announcements]
# 2010/08/05/google-wave%E5%B0%B1%E8%A6%81%E8%AF%B4%E6%8B%9C%E6%8B%9C%E4%BA%86

今天听说Google放弃Wave这个产品了，真是可惜。我还是挺喜欢这个产品的，它比Email功能强大多了。

https://blog.qizhen.xyz/2010/08/05/google-wave%E5%B0%B1%E8%A6%81%E8%AF%B4%E6%8B%9C%E6%8B%9C%E4%BA%86

**Comment by ruanqizhen** [comment]
之前博客下的留言：

Jucao
2010年08月6日 12:19 编辑
Google Wave用了一下，真的不知道它要做什么。像我们这样的技术人员都搞不清楚，怎么指望它能成功呢？呵呵

回复
